src/Neo/SmartContract/Native/CryptoLib.cs

```diff
@@ -63,6 +65,21 @@ public static byte[] Murmur32(byte[] data, uint seed)
             return murmur.ComputeHash(data);
         }
 
+        /// <summary>
+        /// Computes the hash value for the specified byte array using the keccak256 algorithm.
+        /// </summary>
+        /// <param name="data">The input to compute the hash code for.</param>
+        /// <returns>Computed hash</returns>
+        [ContractMethod(CpuFee = 1 << 15, ActiveIn = Hardfork.HF_Manticore)]
```

Comment:
@roman-khimov @Liaojinghui nullable Enums are not valid as param in attributes... Back to HF_Genesis?

Reply:
Too C#-specific for me to handle, but in general I still don't think we need a `Genesis` hardfork and changing constants.